### DISCUSSION TITLE
Coerce textual emojis into actual emojis

### DISCUSSION THREAD

**Discussion by ibrkhalil** [status-im/status-legacy#16660] [General]
# Feature Issue

Convert textual emojis into actual emojis

### User Story

As a user, When I don't want to open the emoji tab and just write the text reference to that emoji .. I want my entered text to coerce to it's emoji equivalent

### Description
<3 to should be ❤️ and so on
![image](https://user-images.githubusercontent.com/33176106/206863933-d96104de-85d1-4fa9-9529-a01d40163f12.png)

**Comment by du82** [comment]
I think this should be a selector like what happens when you try to mention a user. I hate it when apps automatically change what I type into Emojis.

**Comment by ibrkhalil** [comment]
> I think this should be a selector like what happens when you try to mention a user. I hate it when apps automatically change what I type into Emojis.

Even better!
Thanks Du!

**Comment by du82** [comment]
Maybe it should be just like Discord where if I type `:rice:` it shows me a suggestion bar, but if I keep typing and ignore the suggestions, it will leave the text alone
